src/views/AlumniUpdateForm/components/ConfirmationRow/index.js

```diff
@@ -1,8 +1,32 @@
 import { Typography, Grid } from '@material-ui/core/';
 import styles from './ConfirmationRow.module.css';
+import { Check, Clear, Remove } from '@material-ui/icons';
 
 const ConfirmationRow = ({ field, prevValue }) => {
   //CHECKBOX TRUTH VALUE NEEDS TO BE REFORMATTED
+  console.log(field);
+  console.log(prevValue);
+  if (typeof prevValue === 'boolean') {
+    console.log('hi');
+  }
+  const currentValue =
+    typeof prevValue === 'boolean' ? (
+      <Check className={styles.icon_current} />
+    ) : (
+      <Typography variant="subtitle2" className={styles.text_current}>
+        {`${field.value}`}
```

Comment:
This syntax is wonderful 😃 

This is a JSX expression (outermost brackets), with a template string (backticks ``) containing an expression that evaluates to a string.

The simpler way to do this is to put the expression `field.value` as the value of the JSX expression:
```suggestion
        {field.value}
```